lib/jobs/wait-sel-events-job.js

```diff
@@ -0,0 +1,209 @@
+// Copyright 2017, Dell EMC, Inc.
+
+'use strict';
+
+var di = require('di');
+
+module.exports = waitSelEventJobFactory;
+di.annotate(waitSelEventJobFactory, new di.Provide('Job.Wait.Sel.Events'));
+di.annotate(waitSelEventJobFactory, new di.Inject(
+    'Job.Base',
+    'Logger',
+    'Util',
+    'Assert',
+    'Promise',
+    '_',
+    'Services.Waterline'
+));
+
+function waitSelEventJobFactory(
+    BaseJob,
+    Logger,
+    util,
+    assert,
+    Promise,
+    _,
+    waterline
+) {
+    var logger = Logger.initialize(waitSelEventJobFactory);
+
+    /**
+     *
+     * @param {Object} options
+     * @param {Object} context
+     * @param {String} taskId
+     * @constructor
+     */
+    function WaitSelEventJob(options, context, taskId) {
+        WaitSelEventJob.super_.call(this, logger, options, context, taskId);
+        this.options = options;
+        this.nodeId = context.target;
+        this.pollerId = undefined;
+        this.alertFilters = options.alertFilters;
+        this.pollInterval = options.pollInterval || 60000;
+        this.legacyInterval = undefined;
+        this.freshAlerts = [];
+    }
+    util.inherits(WaitSelEventJob, BaseJob);
+
+    /**
+     * @memberOf WaitSelEventJob
+     */
+    WaitSelEventJob.prototype._run = function run() {
+        var self = this;
+        return waterline.workitems.findPollers({
+            node: self.nodeId,
+            'config.command': 'selEntries'
+        })
+        .tap(function(pollers){
+            return self._retrievePollerInfo(pollers);
+        })
+        .then(function(pollers){
+            return self.setSelAlertConfig(pollers[0]);
+        })
+        .then(function(){
+            return self._subscribeSelEvent(self.pollerId, self._callback.bind(self));
+        })
+        .catch(function(err){
+            logger.error('Fail to run wait sel events', {
+                node: self.nodeId,
+                error: err,
+                options: self.options
+            });
+            self._done(err);
+        });
+    };
+
+    /**
+     * @memberOf WaitSelEventJob
+     */
+    WaitSelEventJob.prototype._retrievePollerInfo = function(pollers) {
+        var poller;
+        var existingAlerts;
+        var self = this;
+        if (_.isEmpty(pollers)) {
+            return Promise.reject("Can't find SEL poller");
+        }
+        if (pollers.length !== 1) {
+            return Promise.reject("Find more than one SEL poller for a node");
+        }
+        return Promise.try(function(){
+            poller = pollers[0];
+            self.pollerId = poller.id;
+            self.legacyInterval = poller.pollInterval;
+            existingAlerts = poller.config.alerts || [];
+            _.forEach(self.alertFilters, function(alertFilter){
+                var filter = _.omit(alertFilter, 'count');
+                if (!filter.action || filter.action !== 'critical'){
+                    filter.action = 'critical'; //filter.action will always be critical
```

Comment:
Can this action be "information", and this job only listening to "severity: information" in the sel alert queue? I can't think of a better solution except for changing user's alert config in skupack, but reducing the severity to "information" might reduce the impact to the users. 

Meanwhile, the interface "subscribeSelEvent" in https://github.com/RackHD/on-core/pull/281/files can be more flexible to let the upper layer decide which severity level it would like to pay attention to.